src/app/project/list/list-item-form/list-item-form.component.html

```diff
@@ -23,6 +23,20 @@
     </dsp-string-literal-input>
     <div class="action-bubble" *ngIf="showActionBubble" [@simpleFadeAnimation]="'in'">
         <div class="button-container">
+            <button mat-button
+                    *ngIf="position > 0"
+                    class="reposition up"
+                    title="move up"
+                    (click)="$event.stopPropagation(); repositionNode('up')">
+                <mat-icon>arrow_upward</mat-icon>
+            </button>
+            <button mat-button
+                    *ngIf="!lastPosition"
+                    class="reposition down"
```

Comment:
According to the conventions written [here](https://docs.google.com/document/d/1SIVMxoV0N8xYqhVk_0hhuqSuUF47D1YYrDESqdmipfM/edit#heading=h.9a4ce1jq4p7q) class should be at the end.

Reply:
done in 4f8909d